schemas/compat-data-schema.md

```diff
@@ -275,6 +275,7 @@ For certain browsers, ranged versions are allowed as it is sometimes impractical
   - "≤79" (supported in some version Chromium-based Edge and possibly in EdgeHTML-based Edge)
```

Comment:
I notice this documentation is misleading. It's not in "some" version of Chromium-based Edge, to use this range one has to test Edge 79 specifically.

Same issue with Opera 15 range.

We should probably remove those ranges :)